dispatcher/src/test/java/io/camunda/zeebe/dispatcher/integration/ActorFrameworkIntegrationTest.java

```diff
@@ -125,7 +125,7 @@ protected void onActorStarted() {
     }
 
     void consume() {
-      subscription.poll(this, Integer.MAX_VALUE);
+      actor.run(() -> subscription.poll(this, Integer.MAX_VALUE));
```

Comment:
:question: I don't fully understand how the tasks gets executed after the actor in closed. As far as I understood, the subscription can get woken up, but the task will not execute because the Actor is closed. That said, I think there is no harm in wrapping it in actor.run. 
If this is actually a problem, should we also fix this in LogStorageAppender? We should do either wrap the "onWriteBufferAvailable" in actor#run , or cancel the subscription in LogStorageAppender#onActorClosing.

Reply:
I don't think it can also be woken up either. We traced it together, and I don't think it's possible, but it was clearly still called somehow after the dispatcher was closed, so I figured it doesn't hurt in this case. I wouldn't yet change production code for now.